components/Tabs/README.md

```diff
@@ -291,3 +292,47 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+
+<!-- Extracted from docs/theming-extensions.md -->
+
+### Theming Extensions (Beta)
+
+ `MDCTabBar` supports Material Theming using a Container Scheme. The code is currently
+in Beta readiness, which means you will need to follow the [instructions for adding the
+MaterialComponentsBeta podspec to your
+project](https://github.com/material-components/material-components-ios/blob/73bdc03c2bd2abd032b0b69f05cd76928361aa37/contributing/beta_components.md#beta-program-for-components).
```

Comment:
This link is outdated (it's an older commit). Please consider using either `develop` or d4a2d5d (the latest commit as of this comment).

Reply:
Addressed in 4e8cb3cc3ec6eb9a0b5536dac519b6b866791415